table:
  polygon:
    - [4.900552272796631, 2.867283344268799]
    - [5.435413360595703, 2.719651699066162]
    - [5.305894374847412, 2.242095708847046]
    - [4.807314395904541, 2.411168098449707]
cabinet:
  pose:
    position: 
      x: 4.351056191477719
      y: 1.0972977389006335
      z: 0.0
    orientation: 
      x: 0.0
      y: 0.0
      z: -0.994316154440771
      w: 0.1064677651597742
  shelves:
    second_top:
      name: "second from the top"
      torso_lift_joint: 0.35
      polygon:
        - [2.473784923553467, 0.7884356379508972]
        - [2.803905725479126, 2.1740646362304688]
        - [3.3955090045928955, 2.064265012741089]
        - [3.089816093444824, 0.6275535225868225]
      look_point:
        x: 2.9486193656921387
        y: 1.302085518836975
        z: 1.2523159980773926
      z_min: 1.21
      z_max: 1.48
    second_bottom:
      name: "second from the bottom"
      torso_lift_joint: 0.25
      polygon:
        - [2.473784923553467, 0.7884356379508972]
        - [2.803905725479126, 2.1740646362304688]
        - [3.3955090045928955, 2.064265012741089]
        - [3.089816093444824, 0.6275535225868225]
      look_point:
        x: 2.9666922092437744
        y: 1.2858107089996338
        z: 0.8900102376937866
      z_min: 0.88
      z_max: 1.2
    bottom:
      name: "bottom"
      torso_lift_joint: 0.10
      polygon:
        - [2.473784923553467, 0.7884356379508972]
        - [2.803905725479126, 2.1740646362304688]
        - [3.3955090045928955, 2.064265012741089]
        - [3.089816093444824, 0.6275535225868225]
      look_point:
        x: 2.8490710258483887
        y: 1.3380334377288818
        z: 0.6285148859024048
      z_min: 0.60
      z_max: 0.87

objects:

  # foods
  mayo:
    graspable: false
    category: "foods"
    scale: false
  tuna:
    graspable: false
    category: "foods"
    scale: false
  ketchup:
    graspable: false
    category: "foods"
    scale: false
  oats:
    graspable: false
    category: "foods"
    scale: false
  corn_flour:
    graspable: false
    category: "foods"
    scale: false
  broth:
    graspable: false
    category: "foods"
    scale: false
  broth_box:
    graspable: false
    category: "foods"
    scale: false
  
  # snacks
  peanuts:
    graspable: false
    category: "snacks"
    scale: false
  cornflakes:
    graspable: false
    category: "snacks"
    scale: false
  crisps:
    graspable: false
    category: "snacks"
    scale: false
  pringles:
    graspable: false
    category: "snacks"
    scale: false
  cheese_snack:
    graspable: false
    category: "snacks"
    scale: false
  chocolate_bar:
    graspable: false
    category: "snacks"
    scale: false
  gum_balls:
    graspable: false
    category: "snacks"
    scale: false

  # fruits
  apple:
    graspable: false
    category: "fruits"
    scale: false
  lemon:
    graspable: false
    category: "fruits"
    scale: false
  lime:
    graspable: false
    category: "fruits"
    scale: false
  tangerine:
    graspable: false
    category: "fruits"
    scale: false
  pear:
    graspable: false
    category: "fruits"
    scale: false
  
  # dishes
  spoon:
    graspable: false
    category: "dishes"
    scale: false
  plate:
    graspable: false
    category: "dishes"
    scale: false
  cup:
    graspable: false
    category: "dishes"
    scale: false
  fork:
    graspable: false
    category: "dishes"
    scale: false
  bowl:
    graspable: false
    category: "dishes"
    scale: false
  knife:
    graspable: false
    category: "dishes"
    scale: false
  
  # cleaning supplies
  cloth:
    graspable: false
    category: "cleaning_supplies"
    scale: false
  polish:
    graspable: false
    category: "cleaning_supplies"
    scale: false
  brush:
    graspable: false
    category: "cleaning_supplies"
    scale: false
  sponge :
    graspable: false
    category: "cleaning_supplies"
    scale: false
  
  # drinks
  coffee:
    graspable: false
    category: "drinks"
    scale: false
  kuat:
    graspable: false
    category: "drinks"
    scale: false
  milk:
    graspable: false
    category: "drinks"
    scale: false
  orange_juice:
    graspable: false
    category: "drinks"
    scale: false
  fanta:
    graspable: false
    category: "drinks"
    scale: false
  coke:
    graspable: false
    category: "drinks"
    scale: false
  